docs/src/2.4.ManagingCamera.mdx

```diff
@@ -0,0 +1,170 @@
+import CodeSandboxEmbed from './jsx/utils/CodeSandboxEmbed.js';
+
+# Managing Camera
+
+In this step, you'll learn how to:
+
+- Control camera state
+- Move the camera
+- Update the camera to follow object movements
+
+## Moving the Camera
+
+The WebGL [camera](https://webglfundamentals.org/webgl/lessons/webgl-3d-camera.html) is designed to mimic the way a human eye sees. We've developed a simple and intuitive API that abstracts away much of the math needed to control the camera via `cameraState`. Read the [Camera docs](#/docs/api/camera) to use the playground and learn more.
+
+In this step, let's add a duck model to the center of the scene, and move the camera to observe the duck from different angles.
+
+First, let's create a Duck component using [GLTFScene](#/docs/commands/gltfscene) which we'll use to compose with other command components inside Worldview later. Remember to add the actual `Duck.glb` model file in the `src/utils` folder.
```

Comment:
Inconsistent use of "command" vs "Command". 

Maybe: "First, let's create a Duck component using [GLTFScene](#/docs/commands/gltfscene), which we'll compose with other Command components. Add the `Duck.glb` model file in the `src/utils` folder.